packages/amazonq/test/e2e/amazonq/review.test.ts

```diff
@@ -282,5 +452,157 @@ describe('Amazon Q Code Review', function () {
+                    await execPromise('git config user.name "Test"', { cwd: fileDir })
+                    await execPromise('git config user.email "test@example.com"', { cwd: fileDir })
+                    await execPromise('git commit -m "Initial commit"', { cwd: fileDir })
+
+                    document = await vscode.workspace.openTextDocument(filePath)
+                    await vscode.window.showTextDocument(document)
+
+                    const editor = vscode.window.activeTextEditor
+
+                    if (editor) {
+                        const position = new vscode.Position(20, 0)
+                        await editor.edit((editBuilder) => {
+                            editBuilder.insert(position, '\n')
+                        })
+                        // save file
+                        await document.save()
+                    }
+
+                    // Run the project scan
+                    tab.clickButton(ScanAction.RUN_PROJECT_SCAN)
+
+                    await waitForChatItems(6)
+                    const scanningInProgressMessage = tab.getChatItems()[6]
+                    assert.deepStrictEqual(
+                        scanningInProgressMessage.body,
+                        scanProgressMessage(SecurityScanStep.CREATE_SCAN_JOB, CodeAnalysisScope.PROJECT)
+                    )
+
+                    const scanResultBody = await waitForReviewResults(tab)
+
+                    const issues = extractAndValidateIssues(scanResultBody)
+
+                    assert.deepStrictEqual(
+                        issues.Critical + issues.High + issues.Medium + issues.Low + issues.Info >= 1,
+                        true,
+                        `There are no issues detected when there should be at least 1`
+                    )
+
+                    const uri = vscode.Uri.file(filePath)
+                    const securityDiagnostics: vscode.Diagnostic[] = vscode.languages
+                        .getDiagnostics(uri)
+                        .filter((diagnostic) => diagnostic.source === codewhispererDiagnosticSourceLabel)
+
+                    assertAtLeastOneLLMFindings(securityDiagnostics)
+
+                    const editor2 = vscode.window.activeTextEditor
+                    if (editor2) {
+                        await editor2.edit((editBuilder) => {
+                            const lineRange = editor2.document.lineAt(20).rangeIncludingLineBreak
+                            editBuilder.delete(lineRange)
+                        })
+                        await document.save()
+                    }
+                } finally {
+                    // Clean up git repository
+                    try {
+                        if (isWindows) {
+                            await execPromise('rmdir /s /q .git', { cwd: fileDir })
+                        } else {
+                            await execPromise('rm -rf .git', { cwd: fileDir })
```

Comment:
Why not use `await fs.delete(..., { recursive:true })` ? https://github.com/aws/aws-toolkit-vscode/blob/70ba83fc066cfe8a1f9f50e5eb73a5e1b51d42f4/packages/core/src/shared/fs/fs.ts#L365

 Please always look for shared utils.